performance.lic

```diff
@@ -52,37 +63,64 @@ class Performance
       when /muster the strength to perform/
         pause 30
       when /You begin (.*)/, /You continue (.*)/, /You find yourself relaxing (.*)/, /You concentrate on your sense of rhythm (.*)/, /You cannot use the/
-        exit if done_training?
+        if done_training?
+          wear_hand_armor_and_put_away_instrument
+          exit
+        end
       when /You finish (.*)/
         if done_training?
+          wear_hand_armor_and_put_away_instrument
           exit
         else
-          play_song?(@settings, @song_list, is_instrument_worn, skip_clean)
+          DRC.play_song?(@settings, song_list, is_instrument_worn, skip_clean)
         end
       when /Perhaps you should find somewhere drier/
-        exit
+        if done_training?
```

Comment:
Why is this now checking if `done_training?` - it used to always exit. Is it even possible to keep training when this message is seen?